assets/js/hooks/useAdSenseNotifications.js

```diff
@@ -90,6 +90,11 @@ export default function useAdSenseNotifications() {
 				delete notificationProps.description;
 			}
 
+			// This always shows the `dismissButton` in the `<NotificationFromServer>` layout component.
+			// AdSense alerts are always dismissible, but these will persist only for an hour in
+			// `<NotificationFromServer>`, allowing the alerts to resurface if the issue still persists.
+			notificationProps.dismissible = true;
```

Comment:
It looks like the `isDismissible` attribute is [sent from the server to this notification](https://github.com/google/site-kit-wp/blob/d0da20963155acb841fd757178b7cf0111021c06/includes/Modules/AdSense.php#L397-L405), so why don't we do what we do above with other props instead of always forcing this to be true? eg.:

```js
			if (
				notificationProps.dismissible === undefined &&
				notificationProps.isDismissible !== undefined
			) {
				notificationProps.dismissible = notificationProps.isDismissible;
				delete notificationProps.isDismissible;
			}
```